build.rs

```diff
@@ -3,6 +3,7 @@ fn main() {
     #[cfg(target_os = "macos")]
     {
         println!("cargo:rustc-link-lib=framework=foundation");
+        println!("cargo:rustc-link-lib=framework=security");
```

Comment:
Had to add this to get this working on my M1 -- not sure why it seemed to work for others, although I don't know if most folks use the Makefile (which has this [here](https://github.com/fables-tales/rubyfmt/blob/9ed05fcf5c80c636110fe3430ae9a84ecc425aee/Makefile#L5)).